k8s/aat/cluster-00/pcq/pcq-consolidation-service.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pcq-consolidation-service
+  namespace: pcq
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.job: glob:prod-*
+spec:
+  releaseName: pcq-consolidation-service
+  rollback:
+    enable: true
+  chart:
+    name: pcq-consolidation-service
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    version: 0.0.7
+  values:
+    job:
+      schedule: "*/20 * 31 2 *"
```

Comment:
can u set this to a meaningful cron which doesn't clash other cluster. Current setting means it  will never run , which means it is just for the sake of it.